src/app/series/series.component.ts

```diff
@@ -37,11 +37,20 @@ export class SeriesComponent implements OnInit {
 
   loadSeries() {
     if (this.id) {
-      this.cms.auth.follow('prx:series', {id: this.id}).subscribe(s => {
-        s.follow('prx:account').subscribe(a => {
-          this.setSeries(a, s);
-        });
-      });
+      this.cms.auth.follow('prx:series', {id: this.id}).subscribe(
+        s => {
+          s.follow('prx:account').subscribe(a => {
+            this.setSeries(a, s);
+          });
+        },
+        err => {
+          if (err.status === 404 && err.name === 'HalHttpError') {
```

Comment:
May be because extending `Error` in typescript is a bit wonky in the first place.  Ah well... this works!